include/common/msgType.h

```diff
@@ -26,6 +26,7 @@
 
 #define NAS_RAND_SIZE 16
 #define NAS_AUTN_SIZE 16
+extern int dns_enabled;
```

Comment:
Should be part of mme_config

Reply:
Ok. I will add this in mme_config structure